hack/tools/release/internal/update_providers/README.md

```diff
@@ -0,0 +1,38 @@
+# Update CAPI Providers with CAPI beta releases
+
+`provider_issues` is a go utility intended to open git issues on provider repos about the first beta minor release of CAPI.
+
+## Pre-requisites
+
+- Create a github token with the below access and export it in your environment as `GITHUB_ISSUE_OPENER_TOKEN`. Set the validity to the least number of days since this utility is a one time use tool per release cycle.
+  - `repo:status` - Grants access to commit status on public and private repositories.
+  - `repo_deployment` - Grants access to deployment statuses on public and private repositories.
+  - `public_repo` - Grants access to public repositories
+
+- Export `PROVIDER_ISSUES_DRY_RUN` environment variable to `"true"` to run the utility in dry run mode. Export it to `"false"` to create issues on the provider repositories. Example:
+
+  ```sh
+  export PROVIDER_ISSUES_DRY_RUN="true"
+  ```
+
+- Export `RELEASE_SEMVER` environment variable to the CAPI release version. Only set the major and minor version. The `0-beta.0` is appended by the utility. 
+  Example:
+
+  ```sh
+  export RELEASE_SEMVER="1.6"
+  ```
+
+- Export `RELEASE_DATE` to the targeted CAPI release version date. Fetch the target date from latest [release file](https://github.com/kubernetes-sigs/cluster-api/tree/main/docs/release/releases).
+  Example:
+
+  ```sh
+  export RELEASE_DATE="2023-11-28"
+  ```
+
+## How to run the tool
+
+- Finish the pre-requites [tasks](#pre-requisites).
+- From the root of the project Cluster API, run `make provider-issues` to create issues at provider repositories.
+  - Note that the utility will  
+    - do a dry run on setting `PROVIDER_ISSUES_DRY_RUN="false"` (and will not create git issues)
```

Comment:
I think the true and false may be flipped here

Reply:
Yes, good catch. Fixing it.

Reply:
Fixed in the new force pushed commit. Please take a look again. Thanks!